### ISSUE TITLE
Decred Ambassador certification

### ISSUE THREAD

**Issue by xaur** [xaur/decred-issues#48]
Develop a program to select, train and verify people to represent Decred. They must understand and be able to explain Decred clearly.

The whole program is optional: one is (obviously) not obliged to have a certificate or a title to represent Decred. The intent is to provide some base layer standard of quality for representing the project, so that people know if they reach out to a certified ambassador, they will get a good treatment.

The knowledge can be verified pretty easily during a chat interview. A prepared set of key questions will help.

There can be several levels of knowing and being able to explain Decred, depending on one's technical knowledge. Deep technical knowledge is welcome but is not required. What matters is understanding the fundamental vision and goals of the project.

Aside from knowledge aspect, the person needs to be psychologically stable and not get triggered by things that a lot of today people get increasingly triggered by. Put simply, not be a snowflake, and be able to handle annoying people, trolls, and people with different views on what one think is important in life.

This human aspect is obviously harder to verify than knowledge. The only way I can suggest is that multiple existing ambassadors observe the activity of a candidate over a period of time, say 6 months, and vouch for him. This can be either soft voting or it can employ signing candidate's public key together with something like "he can represent Decred well" message. A web of trust system can be useful here (see #46).

Design a full workflow of bootstrapping, granting and revoking the certificate.

Choose a good name for the certificate, alternatives are: title, clearance, license. It must convey that it is a mark of quality, good attitude, is a mark of respect.

**Comment by xaur** [comment]
Discussed [here](https://matrix.to/#/!MIGqWXfLFBwhipPKYL:decred.org/$1545490750633IcTnV:decred.org) with some serious comments from serious people, and an idea to call the program "Stakey's Angels".

**Comment by xaur** [comment]
Can look at https://ambassadors.rsk.co/

**Comment by xaur** [comment]
Related, from an [article](https://medium.com/@sahand.bagheri/decred-australia-building-a-community-brick-by-brick-89928041687e) by Zohand:

> Communities are also very dependent on their community leaders — especially the first batch of leaders who kickstart everything. These leaders set the tone, the expectations and the standard for whats to come. They also have the responsibility of ensuring the first group of community members are aligned and are committed to participating.
> 
> They make sure to bring others along the journey and mold new leaders who are empowered to contribute and lead. It’s a perpetual cycle. If this is achieved, the community survives indefinitely as long as the business / technology initiative is around. Experienced community leaders do this well.

**Comment by xaur** [comment]
Mentioned in a economic research [proposal](https://proposals.decred.org/proposals/65bde4146b845e7e839d6916d4d8f642bc39c250df5379c2f1e26c4ab778ec1a) from Ammarooni:

> This proposal is to conduct 3 months of economic deep dive to explore the social and economic potential of Decred as a story in human governance and monetary economics. The goal is to create impactful assets which can springboard a network of well educated ambassadors at large.

**Comment by Cagedbird1** [comment]
https://cagedbird1.github.io/decred-Ambassador.io/
I am here based entirely on statistics, not academic. Whoever has more fans and intends to promote decred will be the promotion ambassador of decred

**Comment by Cagedbird1** [comment]
This is a simple model

**Comment by Cagedbird1** [comment]
At the same time, everyone has the opportunity to become an ambassador, because there is also a built-in random function to screen people who participate in promotional activities

**Comment by Cagedbird1** [comment]
What I think is that when people choose to share, they need to record his Twitter account and decred coin account, and then use the crawler program to get the number of fans of each person who submits the sharing task, then store the data, and then display it,and random screening

**Comment by Cagedbird1** [comment]
![Screenshot_2023-03-29-07-22-08-58_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/126668462/228388887-c998fa3a-6988-407f-bcc1-764e9d3db8b0.jpg)
Here you need to pop up a window to remember the currency address and Twitter account, instead of letting them share directly, I will continue to improve!

**Comment by Cagedbird1** [comment]
![Screenshot_2023-03-29-07-22-08-58_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/126668462/228388981-37fb757e-d388-476c-aa59-fc1d593683d8.jpg)

**Comment by Cagedbird1** [comment]
Here you need to pop up a window to remember the currency address and Twitter account, instead of letting them share directly, I will continue to improve!

**Comment by Cagedbird1** [comment]
This code
https://github.com/Cagedbird1/decred-Ambassador.io

**Comment by xaur** [comment]
> Whoever has more fans and intends to promote decred will be the promotion ambassador of decred

We seem to have different meaninigs of "ambassador".

What I was describing is a process like this:

- Experienced community members agree on the definition of "ambassador" and how new ambassadors are "certified" (and "un-certified" if there are major issues). My version is proposed in the [first comment](https://github.com/decredcommunity/issues/issues/48#issue-393679393).
- We start recording who qualifies as an "ambassador" according to the chosen definitions.
- People who qualified and accepted the title can add it to their social media profiles.
- Qualified ambassadors start doing what ambassador is supposed to do - give a quality and accurate representation of the project. Or just continue if they do it already.

In other words, their role is to explain the project, give accurate information, help users with some support issues, and just be nice/not aggressive and not create drama in the process.

Your concept looks like it is a:

- a promotion campaign or system
- success is measured by follower/fan counts reported by the social media platforms (Twtiter, YouTube, etc.)
- best promoters are awarded some DCR

I think we need both but they look different to me.
